aws_topology/tests/json/template/test_elbv2_describe_tags.json

```diff
@@ -1,7 +1,7 @@
 {
     "TagDescriptions": [
         {
-            "ResourceArn": "arn:foo:bar",
+            "LoadBalancerName": "classic-loadbalancer-1",
```

Comment:
I made an incorrect change here - the real problem is that we assumed that elb and elbv2 had the same API response. This is not the case. I will revert this and instead make a new template file